src/menu/NestedMenuItem.jsx

```diff
@@ -24,6 +24,8 @@ var NestedMenuItem = React.createClass({
         menuItems: React.PropTypes.array,
         zDepth: React.PropTypes.number,
         disabled: React.PropTypes.bool,
+        parentItem: React.PropTypes.object,
+        onParentItemClick: React.PropTypes.func,
```

Comment:
Why would we be interested if the parent item was clicked? I would think that the parent would be interested in the child, but not the other way around.